going_postal.js

```diff
@@ -31,149 +31,153 @@ const fs = require('fs')
-		if(!place) return // Last empty line
-
-		if(lastPlace && lastPlace !== place) index++;
-
-		places[place + '_' + index] = places[place + '_' + index] || []
-		places[place + '_' + index].push(number)
-		lastPlace = place
+fetch('https://www.bring.no/radgivning/sende-noe/adressetjenester/postnummer/_/attachment/download/7f0186f6-cf90-4657-8b5b-70707abeb789:676b821de9cff02aaa7a009daf0af8a2a346a1bc/Postnummerregister-ansi.txt')
+	.then(response => {
+	if(!response.ok) throw 'Problems during download of CSV file for postal codes, update URL?'
+	return response.blob()
 	})
-
-	/**
-	 * How much space would it take to represent this place with an if?
-	 * The if takes 33 characters plus the length of the postal place name.
-	 * The Count doesn't matter for how much space the representation will take.
-	 *
-	 * This function doesn't take into account the OR (||) expressions within the
-	 * if.
-	 *
-	 * For instance:
-	 *
-	 * 'if(pn>=0001&&pn<=1295)return "OSLO";'
-	 *
-	 * The if-string for oslo takes 36 characters (minus 3, since the number 1
-	 * only takes one character).
-	 *
-	 * @param C how many postal numbers this place has in this sequence
-	 * @param L lenght of character name of postal place
-	 */
-	function sizeWithIf(C, L) {
-		return 32 + L
-	}
-
-	/**
-	 * How much space would it take to represent this place with object mapping?
-	 * For each postal number C, we need 13 characters plus the name of the place.
-	 *
-	 * For instance:
-	 * {0001: "OSLO", 0010: "OSLO", 0015: "OSLO", ....
-	   *
-	   * This would take (8 + 4) * 639 = 7668 to represent Oslo*.
-	   *
-	   * (* Not really, because I convert numbers to integers. The number of characters
-	   * would be somewhat less for Oslo, but the numbers are true for all other postal
-	   * places.)
-	   *
-	 * @param C how many postal numbers this place has in this sequence
-	 * @param L lenght of character name of postal place
-	 */
-	function sizeWithObject(C, L) {
-		return (8 + L) * C;
-	}
-
-	Object.keys(places).forEach(place => {
-		let numbers = places[place].sort();
-
-		let name = place.split('_')[0]
-
-		// length of character name of postal place
-		let l = name.length
-		// how many postal numbers this place has in this sequence
-		let c = numbers.length
-
-		/*
-		 Should compress the list even more, taking into account that places might
-		 have postal numbers scattered around in the list. But somehow this makes
-		 the final resulting file even larger.
-
-		 This is probably due to that "sizeWithObject" computation fails badly. That
-		 computation doesn't take into account the || expressions within the if.
-
-		 C = (function() {
-			 var tmp = 0
-			 Object.keys(places).forEach(p => {
-				 var newName = place.split('_')[0]
-				 if(name === newName && places[p].length > 2) tmp += places[p].length
+	.then(fileBlob => {
+		console.log('Successfully created retrieved file from Bring!')
+		console.log('File size:', fileBlob.size)
+		console.log('File type', fileBlob.type)
```

Comment:
should console.logs be removed before merge?

Reply:
This file is only used when calling the terminal, so you would get a response stdout with this data, Otherwise, an error would print.

Reply:
ah ok, thats fine